securityconfig/audit.yml

```diff
@@ -0,0 +1,49 @@
+_meta:
+  type: "audit"
+  config_version: 2
```

Comment:
Is there `config_version` 1?

Reply:
7xx have followed config_version 2 for all internal documents in security config. 
6xx have used 1 (or none)